e2e-tests/test/zks-apis.test.ts

```diff
@@ -30,13 +30,14 @@ describe("zks_estimateFee", function () {
     // Act
     const response: Fee = await provider.send("zks_estimateFee", [transaction]);
     // Assert
-    expect(ethers.BigNumber.from(response.gas_limit)).to.eql(ethers.BigNumber.from("3606743"), "Unexpected gas_limit");
+    expect(ethers.BigNumber.from(response.gas_limit).toNumber()).to.be.within(3606743, 5868728, "Unexpected gas_limit");
```

Comment:
why `be.within`?

Reply:
It's the same issue we've discussed in another PR , summarized in https://github.com/matter-labs/era-test-node/issues/320 . I suppose if you're really objecting to a range, we can select the value(s) resulting from the full testsuite run - hopefully they should be stable...

Reply:
```suggestion
    expect(ethers.BigNumber.from(response.gas_limit)).to.eql(ethers.BigNumber.from("4048728"), "Unexpected gas_limit");
```